.github/workflows/publish-scheduled.yml

```diff
@@ -0,0 +1,23 @@
+name: "Scheduled publishing"
+run-name: "Publish to internal - Scheduled"
+
+on:
+    schedule:
+    -   cron: "0 0 * * SUN"
```

Comment:
This will run 3hr prior to the daily scheduled tests. This should be enough time to avoid collisions.